docs/conf.py

```diff
@@ -10,12 +10,44 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
+
 import subprocess
 import sys
 from pathlib import Path
 
-sys.path.insert(0, os.path.abspath("../"))
+import toml
+
+root_dir = Path(__file__).parents[1]
+pyproj = root_dir / "pyproject.toml"
+requirement = root_dir / "docs" / "requirements.txt"
+
+sys.path.insert(0, str(root_dir.resolve()))
+sys.path.insert(0, str((root_dir / "docs" / "tools").resolve()))
+sys.path.insert(0, str((root_dir / "lisa").resolve()))
+
+# -- Requirements ------------------------------------------------------------
```

Comment:
Why can't we just let Poetry handle this?

Reply:
You mean these ``sys.path``-s? Actually I think it would work without all these, I'll try it and if so I will remove them.

Reply:
No, paths are needed, I meant requirements.

Reply:
Alright. My first attempt was to let Poetry handle it, but I ran into an out-of-memory problem on rtd (possibly because of too many packages and too long an installation time), so I had to switch to this solution.